CHANGELOG.adoc

```diff
@@ -37,6 +37,7 @@ Maintenance::
   * Rewrite `AsciidoctorHttpMojoTest` to Java to make it more approachable (#516)
   * Rewrite `AsciidoctorZipMojoTest` to Java to make it more approachable (#518)
   * Rewrite `AsciidoctorDoxiaParserTest` to Java + remove Groovy & Spock configurations (#519)
+  * Replace usage of JRuby `DirectoryWalker` with Java NIO API (https://github.com/stdll[@stdll]) (#532)
```

Comment:
My fault for how I worded the issue :hand: The thing is technically it's an internal "Asciidoctorj" class from the implementation that uses jRuby (under the jruby package), not "JRuby" the library itself. So if we want to support other Asciidoctor implementation in the future, we should not use anything under "asciidoctor.jruby" packages. 
Does this help?

Here is more context https://github.com/asciidoctor/asciidoctorj/issues/1038.